File: Foundation/Linked List/Display.java
public int size(){
      // write code here
      return size;
    }

    public void display(){
      // write code here
      Node temp=head;
      while(temp!=null){
          System.out.print(temp.data+" ");
      temp=temp.next;
    }
        System.out.println();
    }
    
  

File: Foundation/Linked List/Display.cpp
public int size(){
      // write code here
      return size;
    }

    public void display(){
      // write code here
      Node* temp=head;
      while(temp!=NULL){
          cout<<temp->data<<" ";
      temp=temp->next;
    }
        cout<<"\n";
    }
    
  
